.github/workflows/coverage.yml

```diff
@@ -91,6 +91,8 @@ jobs:
         run: make -f .test.mk test-coverage
 
       - name: Upload coverage results to coveralls.io
+        # Avoid interference with the coverage of a submodule.
```

Comment:
Please add a link/description of `inputs.submodule` here or to commit message.

Reply:
Or at least give more details about the problem you want to solve.
Something like: LuaJIT as well as Tarantool uses Coveralls, this will avoid reporting Tarantool's coverage to LuaJIT's namespace in Coveralls.

Reply:
Added to the commit message:

```
ci: don't send coverage results for submodules

If coverage workflow is triggered via submodule as a part of integration
testing, coverage reports from Tarantool are sent to coveralls as
reports from the submodule. Our fork of LuaJIT as well as Tarantool uses
Coveralls, this will avoid reporting Tarantool's coverage to LuaJIT's
namespace in Coveralls.

To avoid this misbehaviour, run the "upload report" step only when there
is no submodule to bump (specified as `inputs.submodule` [1]) as a part
of the reusable workflow.

[1]: https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#example-specifying-inputs

NO_DOC=CI
NO_TEST=CI
NO_CHANGELOG=CI
```

Reply:
Discussed in a private conversation: `inputs.submodule` defined in:

```
  test-tarantool-release_lto_clang:
    name: release lto clang
    uses: tarantool/tarantool/.github/workflows/release_lto_clang.yml@master
    with:
      submodule: luajit
      revision: ${{ github.sha }}
```